README.md

```diff
@@ -1,9 +1,47 @@
+## PCL Mean and Covariance
+
 Experiments with mean and covariance computation in PCL.
 
-1. Clone this repository with submodules (pass `--recursive` flag)
+## Installation
+
+1. **Clone this repository** 
+
+   Clone with submodules and pass the `--recursive` flag:
+
+       $ git clone https://github.com/taketwo/pcl-mean-and-covariance.git --recursive-submodules
```

Comment:
`--recursive` and `--recursive-submodules` are synonyms, so you are free to use either one, but do so consistently (which is currently not the case, textual description and command use different flags).

Reply:
Ah, thanks for bringing me up on that. It appears that the `--recursive` flag is depreciated in favour of `--recursive-submodules`, hence why I changed it.